tests/time_tests/common/statistics_writer.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <sstream>
+#include <fstream>
+#include <cstdio>
+
+
+class StatisticsWriter {
```

Comment:
Could you add a little bit more comments to the code? What is the statistics format? Yaml?

Reply:
Ok, I will add a docstrings. Format: `<step_name>:<duration in float>\n`

Reply:
I added docstrings. Current format is YAML compatible (reading with python `yaml` will convert it to dictionary like `{step_name: duration}`)